gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go

```diff
@@ -264,6 +264,7 @@ func (p *glusterBlockProvisioner) Provision(options controller.VolumeOptions) (*
 		modeAnn = "url:" + cfg.url + "," + "user:" + cfg.user + "," + "secret:" + cfg.restSecretName + "," + "secretnamespace:" + cfg.restSecretNamespace
 	}
 
+	volMode := v1.PersistentVolumeBlock
```

Comment:
this should come from PersistentVolumeClaimSpec.VolumeMode in volumeOptions

Reply:
@rootfs I thought this will be always `block` , so hard coding to `block` is good enough. Are you saying it can be chosen by user by defining a different mode in PVC ?

Reply:
Yeah, true , as these are iscsi volumes it can go with any of it.. I will make the change.